healthmgr reports null pointer error

The `getMetricValueSum` may return null, which needs validation before use the return value.
```
[2017-08-29 14:56:05 -0700] [STDERR] stderr: Exception in thread "main"   
[2017-08-29 14:56:05 -0700] [STDERR] stderr: java.util.concurrent.ExecutionException: java.lang.NullPointerException  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.HealthManager.main(HealthManager.java:218)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: Caused by: java.lang.NullPointerException  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.sensors.BackPressureSensor.get(BackPressureSensor.java:89)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.detectors.BackPressureDetector.detect(BackPressureDetector.java:59)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.microsoft.dhalion.policy.HealthPolicyImpl.lambda$executeDetectors$3(HealthPolicyImpl.java:97)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.microsoft.dhalion.policy.HealthPolicyImpl.executeDetectors(HealthPolicyImpl.java:99)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at com.microsoft.dhalion.policy.PoliciesExecutor.lambda$start$1(PoliciesExecutor.java:52)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)  
[2017-08-29 14:56:05 -0700] [STDERR] stderr: 	at java.lang.Thread.run(Thread.java:745)  

```

--- a/heron/executor/src/python/heron_executor.py
+++ b/heron/executor/src/python/heron_executor.py
@@ -449,7 +449,7 @@ class HeronExecutor(object):
                      "--cluster", self.cluster,
                      "--role", self.role,
                      "--environment", self.environ,
-                     "--topology_name", self.topology_name]
+                     "--topology_name", self.topology_name, "--verbose"]
 
     return healthmgr_cmd
 

--- a/heron/healthmgr/src/java/com/twitter/heron/healthmgr/sensors/BackPressureSensor.java
+++ b/heron/healthmgr/src/java/com/twitter/heron/healthmgr/sensors/BackPressureSensor.java
@@ -18,6 +18,7 @@ package com.twitter.heron.healthmgr.sensors;
 import java.time.Duration;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.logging.Logger;
 
 import javax.inject.Inject;
 
@@ -32,6 +33,8 @@ import com.twitter.heron.healthmgr.common.TopologyProvider;
 import static com.twitter.heron.healthmgr.sensors.BaseSensor.MetricName.METRIC_BACK_PRESSURE;
 
 public class BackPressureSensor extends BaseSensor {
+  private static final Logger LOG = Logger.getLogger(BackPressureSensor.class.getName());
+
   private final MetricsProvider metricsProvider;
   private final PackingPlanProvider packingPlanProvider;
   private final TopologyProvider topologyProvider;
@@ -86,7 +89,11 @@ public class BackPressureSensor extends BaseSensor {
         // manager instance in the result
         InstanceMetrics stmgrInstanceResult = streamManagerResult.values().iterator().next();
 
-        double averageBp = stmgrInstanceResult.getMetricValueSum(metric) / duration.getSeconds();
+        Double valueSum = stmgrInstanceResult.getMetricValueSum(metric);
+        if (valueSum == null) {
+          continue;
+        }
+        double averageBp = valueSum / duration.getSeconds();
 
         // The maximum value of averageBp should be 1000, i.e. 1000 millis of BP per second. Due to
         // a bug in Heron (Issue: 1753), this value could be higher in some cases. The following

